### DISCUSSION TITLE
Add other models index page to Create form page

### DISCUSSION THREAD

**Discussion by Cresta47** [Laravel-Backpack/community-forum#663] [Q&A (Help)] [answered]
What we are  trying to achieve is
We have the Booking Model, and I should be able to insert other pages index table to Booking create form
We need other models table inside this Booking form
We need to have the features sames as in the list(index table), search,clone, inline edit, bulk delete and all the features.
We should be able to add other models and append to that table.
So is it possible to add these features from other model in this Booking (model) create table?

**Comment by promatik** [comment, marked as answer]
Hi @Cresta47, I'm struggling to understand what you need...

So, you have a Booking CRUD, where you have a table with Bookings. And while creating a new Booking, you want to have access to other CRUDs data tables, is it right?

For this, I don't believe there is a way to do it, and I don't think it is practical in terms of UI/UX.
What I recommend you is to create a widget where you display those tables/entries in a succinct way, in order for it to fit the form, both ui and ux.

> _We should be able to add other models and append to that table._

Regarding this you can take a look at [InlineCreate Operation](https://backpackforlaravel.com/docs/6.x/crud-operation-inline-create).
